src/mtool/api/rest/rest_api/dagent/metrics.py

```diff
@@ -251,15 +238,11 @@ def get_vol_read_latency(time, arr_ids, vol_ids, auth = BASIC_AUTH_TOKEN):
     except Exception as err:
         print(f'Other error occurred: {err}')
 
-def get_write_latency(time, arr_ids, auth = BASIC_AUTH_TOKEN):
-    req_headers = get_headers(auth)
+def get_write_latency(metric_query_path, auth = BASIC_AUTH_TOKEN):
     try:
-        #print("Sending command to dagent")
-        PATH = ARRAY_PATH.format("writelatency",arr_ids,time)
-        response = send_command_to_dagent("GET",url = PATH, headers= req_headers,timeout=(connect_timeout,read_timeout))
-        #print("--------------RESPONSE-------------")
-        response = response.json()
-        #print("--------------RESPONSE-------------")
+        PATH = "{metric_query_path}{walv}".format(metric_query_path=metric_query_path, walv=WRITE_AVG_LAT_VOLUME)
+        response = requests.get(PATH)
+        response = json.loads(response.content)
```

Comment:
Please check the scenario if Telemetry is not running then how requests.get() and json.loads() will behave. 
Will this function return meaningful error message to UI ?

Reply:
Added is Telemetry Endpoint up check when loading dashboard page, showed alert popup if not running with meaningful error message.